oragono.yaml

```diff
@@ -415,6 +415,9 @@ limits:
     # nicklen is the max nick length allowed
     nicklen: 32
 
+    # identlen is the max ident length allowed
+    identlen: 10
```

Comment:
(if we up the default limit, we should up this one to match)

Reply:
(upped to 20 as well)